lib/DDG/Spice/JrDevJobs.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::JrDevJobs;
+# ABSTRACT: Search for Junior Developer Jobs from JrDevJobs.com
+
+use DDG::Spice;
+
+triggers start => qw(jrdevjobs);
+
+spice to => 'https://www.jrdevjobs.com/jobs.json?query=$1';
+spice is_cached => 1;
+spice wrap_jsonp_callback => 1;
+
+handle remainder => sub {
```

Comment:
You probably want to use `remainder_lc` because your regex below are case-sensitive.

Alternatively you could use `s/(jobs?|developer)//i` which forces case-insensitive regex matching